test/integration/globals/app-nav.js

```diff
@@ -48,8 +48,12 @@ context('App Nav', function() {
       .routeFlows()
       .routePrograms()
       .routeDashboards()
-      .visit();
+      .visit()
+      .then(() => {
+        const storageItem = JSON.parse(localStorage.getItem('isNavMenuMinimized'));
 
+        expect(storageItem).to.be.false;
+      });
```

Comment:
Verifies that the `localStorage.isNavMenuMinimized` value is set to `false` by default if the local storage item doesn't exist yet.